cli/cmd/common.go

```diff
@@ -50,6 +51,7 @@ func waitForFinish(ctx context.Context, cancel context.CancelFunc) {
 }
 
 func openBrowser(url string) {
+	go socket.OpenWebsocket(url)
```

Comment:
why do we need websocket from cli?

Reply:
@IgorGov I just didn't revert the code related to streaming the events from CLI -> API Server -> Web UI such that the desired web UI behavior can be implemented later on. Related https://github.com/up9inc/mizu/pull/304#issuecomment-927976401